public/app/plugins/panel/nodeGraph/Node.tsx

```diff
@@ -152,10 +161,27 @@ function NodeContents({ node, hovering }: { node: NodeDatum; hovering: HoverStat
     return null;
   }
 
+  const iconSize = node.iconSize?.values[node.dataFrameRowIndex] || iconS;
```

Comment:
Should we do any runtime validation with a fallback that the dataFrame contains  a valid iconSize?